test/tests/inittest.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# This have to be first test for cockpit
+#  install build dependencies
+#  download new version of cockpit from git 
+#  compile it
+# it is basic test compilation must PASS
+
+set -x
+
+BUILD="build1"
+PACKAGE="cockpit"
+SOURCE="https://github.com/$PACKAGE-project/$PACKAGE.git"
+
+#
+yum-builddep -y $PACKAGE
```

Comment:
Thanks,
It is nice and clear solution.
Probably it would be also repaired in HACKING.md:
now there is: 
$ srpm=$(tools/make-srpm)
$ sudo yum-builddep $srpm
$ sudo yum install nodejs npm

but these instruction from some reason doesn't work for me, First variable srpm contains "cockpit-ws" value and next command failed. Probably I've done something wrong, or it seems that there was some changes.